services/api-server/src/simcore_service_api_server/services/webserver.py

```diff
@@ -351,7 +351,7 @@ async def get_projects_w_solver_page(
             show_hidden=True,
             # WARNING: better way to match jobs with projects (Next PR if this works fine!)
             # WARNING: search text has a limit that I needed to increase for the example!
-            search=urllib.parse.quote(solver_name, safe=""),
+            search=solver_name,
```

Comment:
hahaha!  I would add a test that catches this next time?  what library is the culprit!